content-metadata-protobuf/proto/Channel.proto

```diff
@@ -1,5 +1,7 @@
 syntax = "proto2";
 
+package muthu.other;
```

Comment:
Also - I'm not sure what is the reason behind this change. Is this the reason why there are so many diffs in compiled protobuffs?

Reply:
libprotoc 3.6.1

Reply:
Thanks. I think the buffers were previously compiled using a newer version. I'm currently using `libprotoc 3.15.6` and I'm getting results similiar to those from before this PR. Perhaps this changes can be reverted?

Reply:
Jsut compiled them with libprotoc 3.15.6, the diffs looks the same now, will update PR shortly after resolving `yarn.lock` issues